src/gatsby-theme-docz/components/Playground/index.js

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable react/prop-types */
```

Comment:
I'm assuming most of this is direct from the main Docz theme?

Reply:
Yep, only thing i really changed in here was adding the `wrapper` class and changed the margin in the styles.